src/ps_collector/parsePush.py

```diff
@@ -355,10 +357,9 @@ def parse_push(self, parsed_object: dict):
                 int(timestamp): int(throughput)
             }
         elif test_type == "trace":
-            # Traceroute data is wrapped in an extra array
-            to_return['datapoints'] = {
-                int(timestamp): parsed_object['result'][self.topic_map[test_type]['datapoint']][0]
-            }
+            path = copy.copy(parsed_object['result']['paths'][0])
+            self.trace_path_filter(path)
```

Comment:
I don't think you call a static method like this.  I don't think you use `self`, I think you use the class name.

Reply:
Good catch! I'll get that fixed.